src/PathTracedVolume.ts

```diff
@@ -636,13 +547,14 @@ export default class PathTracedVolume implements VolumeRenderImpl {
     this.resetProgress();
   }
 
-  updateActiveChannels(image: VolumeDrawable): void {
+  updateActiveChannels(channelColors: FuseChannel[], channelData: Channel[]): void {
     const ch = [-1, -1, -1, -1];
     let activeChannel = 0;
     const NC = this.volume.num_channels;
     const maxch = 4;
     for (let i = 0; i < NC && activeChannel < maxch; ++i) {
-      if (image.isVolumeChannelEnabled(i) && image.getChannel(i).loaded) {
+      // check that channel is not hidden (rgbColor !== 0) and is loaded
+      if (channelColors[i].rgbColor !== 0 && channelData[i].loaded) {
```

Comment:
Also optional, minor comment:  I feel like `FuseChannel` should just have a member function `enabled():bool { return this.rgbColor !== 0; }` to hide this detail.  Perhaps using a named symbolic constant instead of literal 0 would help.  `const FUSE_DISABLED = 0;` or something.